### PR TITLE
Exposed getPreviousVersion() API

### PR DESCRIPTION
**Pull request by mockitoguy**
Hey team! Would this new API make sense?

It is useful to expose the previous version for automated generation of the changelog. This has been requested before in #138. Sample usage of this new property here: https://github.com/mockito/mockito-kotlin/pull/421

Thoughts?

### CODE REVIEW

**Comment by mockitoguy** [comment]
@bgalek, thoughts?

**Comment by bgalek** [comment]
@mockitoguy nice and small, LGTM :)

File: src/main/groovy/pl/allegro/tech/build/axion/release/domain/VersionConfig.groovy [hunk 1]
```diff
@@ -152,6 +152,12 @@ class VersionConfig {
         return resolvedVersion.decoratedVersion
     }
 
+    @Nested
+    String getPreviousVersion() {
+        ensureVersionExists()
+        return resolvedVersion.previousVersion
```
Comment (bgalek) [inline src/main/groovy/pl/allegro/tech/build/axion/release/domain/VersionConfig.groovy:158]: could you extend at least one test with this filed verified?

**Comment by bgalek** [comment]
@mockitoguy little nitpicks and then I think we can merge it :)

**Comment by mockitoguy** [comment]
@bgalek, can you review? Thanks for feedback!

**Review by bgalek — APPROVED** [review]

**Comment by bgalek** [comment]
@mockitoguy nice! thx! Last thing, docs update ;) it could be in next PR but we need it before release :) I would like to release new version this weekend ;)

**Comment by o-shevchenko** [comment]
Hey
That's very useful API.
It would be super helpful to have Gradle task for that added into the plugin.
```
./gradlew previousVersion
```

**Pull request merged**